cmd/modern/root_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package main
+
+import (
+	"github.com/microsoft/go-sqlcmd/internal/cmdparser"
+	"github.com/microsoft/go-sqlcmd/internal/cmdparser/dependency"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// TestRoot is a quick sanity test
+func TestRoot(t *testing.T) {
+	c := cmdparser.New[*Root](dependency.Options{})
+	c.DefineCommand()
+	c.SetArgsForUnitTesting([]string{})
+	c.Execute()
+}
+
+func TestIsValidSubCommand(t *testing.T) {
+	c := cmdparser.New[*Root](dependency.Options{})
+	invalid := c.IsValidSubCommand("nope")
+	assert.Equal(t, false, invalid)
+	valid := c.IsValidSubCommand("query")
+	assert.Equal(t, true, valid)
+}
```

Comment:
using Equalf with a message would improve logging. If a test fails it'd be nice to know which assertion was failing without having to get the line number and looking up the code. Line numbers are ephemeral too so if you are looking at a test failure you have to know which branch/commit it was synced to.